app/controllers/discover.js

```diff
@@ -323,6 +333,11 @@ export default Ember.Controller.extend(Analytics, {
             }
         },
 
+        loadPage(pageNumber) {
+            this.set('page', pageNumber);
+            this.loadPage();
```

Comment:
Having an action with the same name as a method can be very confusing. Consider calling the latter `this._loadPage`, or somesuch.